.github/workflows/checks.yml

```
@@ -2,91 +2,310 @@ on:
   workflow_call:
     inputs:
       push_coverage:
-          type: boolean
-          description: "If true, the coverage results are pushed to codecov.io."
-          default: true
+        type: boolean
+        description: "If true, the coverage results are pushed to codecov.io."
+        default: true
     secrets:
       CODECOV_TOKEN:
-        required: false        
+        required: false
 
 env:
-  HF_HUB_DOWNLOAD_TIMEOUT: "60"
-  HF_HUB_ETAG_TIMEOUT: "60"
+  HF_HUB_DOWNLOAD_TIMEOUT: "90"
+  HF_HUB_ETAG_TIMEOUT: "90"
   UV_FROZEN: "1"
+  PYTEST_ML: |-
+    tests/test_e2e_conversion.py
+    tests/test_e2e_ocr_conversion.py
+    tests/test_backend_webp.py
+    tests/test_asr_pipeline.py
+    tests/test_threaded_pipeline.py
+  PYTEST_TO_SKIP: |-
+    tests/test_e2e_conversion.py::test_e2e_pdfs_conversions
```

we should not skip the main test

```suggestion
  PYTEST_TO_SKIP: ""
```